Resources/translations/maintenance.nl.yml

```diff
@@ -0,0 +1,7 @@
+lexik_maintenance.success_unlock: Server is online.
```

Comment:
This line is not translated

Reply:
The Dutch translation is the same as the English one.